src/app/archive_hardfork_toolbox/sql.ml

```diff
@@ -132,3 +132,19 @@ let fetch_latest_migration_history_query =
 
 let fetch_latest_migration_history (module Conn : CONNECTION) =
   Conn.find_opt fetch_latest_migration_history_query ()
+
+let fetch_last_filled_block_query =
```

Comment:
Could you explain&document the logic here? We're looking at the first block after the last block with no txns it seems.